docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -64,13 +72,9 @@ The extra step of creating an app project just to get complete warnings for a li
 ```
 
 ```dotnetcli
-dotnet publish -c Release
+dotnet publish -c Release -r <RID> -f net6.0
```

Comment:
For the example, I would set the RID either in the project file, or on the command-line, not both. Maybe we could point to the self-contained publish docs https://docs.microsoft.com/en-us/dotnet/core/deploying/deploy-with-cli#self-contained-deployment if you think it's worth clarifying.

I'd suggest setting `RID` on the command-line only, and the TFM in the project file only, to match those docs.